src/icon-tab-bar.scss

```diff
@@ -751,6 +781,27 @@ $fd-icon-tab-bar-semantic-values: (
 
         font-size: 1rem;
       }
+
+      @each $set-name, $set-params in $fd-icon-tab-bar-semantic-values {
+
+        &--#{$set-name} {
+          @include fd-icon-tab-bar-list-item-color(map_get($set-params, 'color'));
+        }
+      }
+    }
+
+    .#{$block}__list-item-icon-container {
+      @include fd-flex-vertical-center();
+      @include fd-set-margin-right(0.5rem);
+
+      width: 1.25rem;
+    }
+
+    // Need to rewrite styles for fd-list__icon
+    .#{$block}__list-item-icon [class*=sap-icon],
+    .#{$block}__list-item-icon [class*=sap-icon]:last-child {
+      margin: 0 !important;
+      width: auto !important;
```

Comment:
Can't you use specificity to avoid relying on !important?

Reply:
Yes. Strange, I tested it yesterday and fd-list__icon has higher priority. But today I have recheck and it works well without important.
Removed important